### DISCUSSION TITLE
Welcome to Local_Seva Discussions!

### DISCUSSION THREAD

**Discussion by sakshgupta** [sakshgupta/Local_Seva#29] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YuZhangWang** [comment, marked as answer]
Congratulations on the new discussion feature, which provides a place for developers to communicate **more easily and quickly** than the issues feature!

In my opinion, this feature is a bit like a discord-like platform that allows people who use the same repository to communicate together, rather than simply suggesting bugs or improving features.

More, for my part, I know of an interesting application for discussion boards.The comment section of any of the repositories can be used as the comment function of a website, the specific implementation can be referred to the following two repositories, **[giscus](https://github.com/giscus/giscus) and [utterances](https://github.com/utterance/utterances)**.

Here are some specific application results, I am currently using it more convenient, because after this operation, you can not build on the server comment function, which makes my personal blog, become **more lightweight** and at the same time **more functional**!

![图片](https://github.com/sakshgupta/Local_Seva/assets/65295250/da9248ec-34ba-49ed-bdcf-ea95f6c6843e)
![图片](https://github.com/sakshgupta/Local_Seva/assets/65295250/b738ed3c-f9ae-4fe8-b360-3866fffaca22)
